src/containers/entitysetsearch/EntitySetSearchResults.js

```diff
@@ -599,6 +599,14 @@ export default class EntitySetSearchResults extends React.Component {
     });
   }
 
+  removeDuplicates = (list1 :List<any>, list2 :List<any>) => {
+    const valueSet = new Set();
+    list1.concat(list2).forEach((val) => {
+      valueSet.add(val);
+    });
+    return Immutable.List(valueSet);
```

Comment:
If you use `Immutable.Set()` above, you can then just do `return valueSet.toList()`. But, do you want a `List` or a `Set`? Does it matter if it's a `List` or a `Set` here? If it doesn't matter, just return `valueSet` itself.